src/main/java/uk/gov/companieshouse/account/validator/service/FilesStorageServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.companieshouse.account.validator.service;
```

Comment:
Why is this service storing files locally?

Reply:
built in this way as a first example
Here we have the Exception when trying to upload an existing file. 
I will change to another (in memory) mechanism to block or identify if the files have been uploaded twice, or is already in progress.
I need to work this out